FormattingStyleGuide.md

```diff
@@ -0,0 +1,43 @@
+# Formatting & Style Guide
+
+## Scope and purpose
+This document provides formatting and style conventions for publishing documents in __GitHub__ in __Markdown__ format. Some advice on document conversion and how to meet more sophisticated publishing needs is offered.
+
+## General
+- Use [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/).
+- If the workflow prescribes the document to go into BitBucket repository first, then you may find [BitBucket Markdown Syntax Guide](https://confluence.atlassian.com/bitbucketserver/markdown-syntax-guide-776639995.html) useful.
+
+## Title & headings
+- Use # (h1) for document title.
+- Use ## (h2) for top-level headings.
+- Number the headings manually. There's no heading numbering support in Markdown.
```

Comment:
Yes, this is referencing by heading title. For technical documentation, however, we often need shorter format references. For example, to reference individual test cases in a expansive document, heading numbering is essential. Also for protocols - where's clearly articulated requirements, numbering is very useful (cf legal acts).

I also want to note that choice of the tools must should be guided by practice, not vice versa. Now this is a difficult situation. Note that Confluence has closed format, we - as well as you - use confluence as well. I propose that each document should be evaluated individually. For really important documents, I'd invest my time to do numbering by hand.

"Heading numbering requires manual manual work and is advised only for documents where there is special need or benefit of it"?